CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to BLOM/iHAMOCC
+We welcome contributions to the BLOM/iHAMOCC model, including bug reports and fixes, contributions for code improvements, and suggestions for feature enhancements or inclusion of new features.
+
+The latest version of the BLOM/iHAMOCC source code is hosted on github:
+https://github.com/NorESMhub/BLOM
+
+We encourage using the [issue ticket](https://github.com/NorESMhub/BLOM/issues) system in github to get in touch with the model developers regarding any questions or suggestions you might have.
+
+BLOM/iHAMOCC is the ocean circulation model and ocean BGC model component of the [NorESM](https://github.com/NorESMhub/NorESM) model system. The most common use of BLOM/iHAMOCC is as an integrated part of NorESM, and the documentation for BLOM/iHOMACC is therefore also integrated with the [NorESM documentation](https://noresm-docs.readthedocs.io/en/latest/index.html). Please feel free to drop us a note through the issue tracker if you don't find answers to your questions there.
+
+## Reporting bugs
+If you find bugs, strange output or something puzzling, please raise an issue. No issue is too big nor too small. Before creating a new issue, we would be grateful if you first check existing issues in case the bug has already been reported. In this case you may want to add your comments to the existing bug report.
+
+Please consider providing additional information with your bug report, e.g.
+- model version for BLOM/iHAMOCC and/or NorESM
+- HPC platform and environment settings
+- Model settings (compset, grid resolution, etc.)
+- Error messages, log output
+
+## Suggesting new features or feature enhancements
```

Comment:
I wonder whether we should add something about discussing new features with the relevant members of the core BLOM/iHAMOCC group? If someone wishes to implement a new feature this has to be discussed beforehand to ensure that we actually want to include this feature and to provide some guidance on how to implement this best.

Reply:
Yes, I agree. I will add something about this.